build/ipdk

```diff
@@ -0,0 +1 @@
+scripts/ipdk.sh
```

Comment:
The install line (line 64) in the README_DOCKER.md can be changed back to `./ipdk install` instead of `./scripts/ipdk.sh install` 😉

Reply:
Good catch!